content/faq/dmca.md

```diff
@@ -10,20 +10,20 @@ The Digital Millennium Copyright Act (DMCA) is United States Copyright Law writt
 
 ### How does LBRY handle and respond to a DMCA request or takedown notice?
 
-Upon receipt of a DMCA request from the content owner or legal representative, LBRY Inc will use its best judgment to determine validity. We will block access to content deemed infringing in any LBRY Inc. owned applications. LBRY Inc. will also remove any data related to the infringing content from servers within its control. 
+Upon receipt of a DMCA request from the content owner or legal representative, LBRY Inc. will use its best judgment to determine validity. We will block access to content deemed infringing in any LBRY Inc. owned applications. LBRY Inc. will also remove any data related to the infringing content from servers within its control.
 
-Due to the immutable nature of the LBRY blockchain, a record and metadata of the infringing content may continue to exist if the original publisher does not remove it. The data may still exist on the original publisher's computer (and anyone who may have downloaded it prior to it being blocked) but will not be accessible through any LBRY Inc controlled applications.  
+Due to the immutable nature of the LBRY blockchain, a record and metadata of the infringing content may continue to exist on network hosts if the original publisher does not remove it. The data may still exist on the original publisher's computer (and anyone who may have downloaded it prior to it being blocked) but will not be accessible through any LBRY Inc. controlled applications.  
 
 ### Where are DMCA requests recorded?
 
-You can view all of the DMCA requests LBRY receives on our [DMCA GitHub Repository](https://github.com/lbryio/dmca), categorized by year and dated by filename. 
+You can view all of the DMCA requests LBRY receives on our [DMCA GitHub Repository](https://github.com/lbryio/dmca), categorized by year and dated by filename.
 
 ### Can I dispute a DMCA complaint?
 
-Yes, you may. We can't give you legal advice, so check with your local attorney or legal group about how to do this. There are also good resources online to learn about DMCA counter-notices. The [EFF has published an excellent guide](https://www.eff.org/issues/intellectual-property/guide-to-youtube-removals) about how to deal with legal issues facing online content creators. If you have done the research or feel the content was blocked in error, [please reach out to us](mailto:help@lbry.io). 
+Yes, you may. We can't give you legal advice, so check with your local attorney or legal group about how to do this. There are also good resources online to learn about DMCA counter-notices. The [EFF has published an excellent guide](https://www.eff.org/issues/intellectual-property/guide-to-youtube-removals) about how to deal with legal issues facing online content creators. If you have done the research or feel the content was blocked mistakenly, [please reach out to us](mailto:help@lbry.io). 
```

Comment:
I think "in error" fits better in this legal context.

Reply:
This specific section "Can I dispute a DMCA complaint" (I don't think) is being delivered in a legal context, it's more of a disclaimer for advisory I would say.

Reply:
'Unfairly' might work. To me, mistakenly sounds like one of those things you can technically do in English, but that sounds funny to me.

Reply:
I feel like mistakenly would be better suited in terms of legal context, unfairly sounds more like something that was censored by administrators **not** in the context of legality.